### PR TITLE
Objects Automated Test - LPS-162140 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-162140

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro [hunk 1]
```diff
@@ -40,6 +40,8 @@ definition {
 	macro addPicklistItemViaUI {
 		SelectFrame(locator1 = "Picklist#IFRAME_PICKLIST_MODAL");
 
+		WaitForPageLoad();
+
 		Click.javaScriptClick(locator1 = "Picklist#ADD_ITEM_BUTTON");
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro:46]: In the future, please add this in a separate commit. It's unclear why we need `WaitForPageLoad();` here

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path [hunk 2]
```diff
@@ -119,7 +119,7 @@
 	</tr>
 	<tr>
 		<td>TYPE_OPTION</td>
-		<td>//button[@class='dropdown-item']//div[contains(text(),"${key_fieldType}")]</td>
+		<td>//button[@class='dropdown-item']//div[(text()="${key_fieldType}")]</td>
 		<td></td>
 	</tr>
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path:124]: If we're making changes to a path, we should be doing this in a separate commit with a clear commit message explaining why. I'm only expecting a new automated test and new macros/paths for the test in this commit.

@andreefarias please follow this convention from now on, I'll be enforcing this in the future.

**Review by timpak — APPROVED** [review]
Good job with the test! Please see my comments for conventions to follow from now on.

**Comment by timpak** [comment]
LGTM! See https://github.com/timpak/liferay-portal/pull/308

**Pull request closed without merging**